<launch>
<node name="dt_target_object_node" pkg="dt_target_object" type="dt_target_object_node" output="screen">
                                         <!--packing-->
      <param name="SourceID" value="11"/>
      <param name="TargetID" value="-1"/> <!--(-1): publish to everyone-->
      <param name="DtObjectID" value="-1"/>
      <param name="CloudMessagePubTopic" value="/R_UAV_0/Message_to_Cloud"/>
      <param name="LocalPositionMessageSubTopic" value="/mavros/local_position/pose"/>
      <param name="MessagePubFrequency" value="10"/>

                                         <!--unpacking-->
      <param name="CloudMessageSubTopic" value="/R_UAV_0/Message_from_Cloud"/>
      <param name="TargetVelocityMessagePubTopic" value="/R_UAV_0/Target/Velocity"/>

      <param name="StartPositionX" value="1"/>
      <param name="StartPositionY" value="1"/>
      <param name="StartPositionZ" value="0"/>

</node>
</launch>
